### PR TITLE
Classifier: localise tutorials

### PR DESCRIPTION
**Pull request by eatyourgreens**
- Pass translation `strings` down to tutorial components.
- Use the `strings` dictionary to show step content.

## Package
lib-classifier

## How to Review

Load PH-TESS in the dev classifier and change the language.
https://localhost:8080/?project=nora-dot-eisner/planet-hunters-tess&env=production

https://user-images.githubusercontent.com/59547/169840507-0f3cb79f-28cf-46bb-9e46-3d7932135dac.mov

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General
- [x] Tests are passing locally and on Github
- [x] Documentation is up to date and changelog has been updated if appropriate
- [x] You can `yarn panic && yarn bootstrap` or `docker-compose up --build` and FEM works as expected
- [ ] FEM works in all major desktop browsers: Firefox, Chrome, Edge, Safari (Use Browserstack account as needed)
- [ ] FEM works in a mobile browser

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
- [x] All pages of a FEM project load: Home Page, Classify Page, and About Pages
- [x] Can submit a classification
- [x] Can sign-in and sign-out
- [x] The component is accessible
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## New Feature
- [x] The PR creator has listed user actions to use when testing the new feature
- [ ] Unit tests are included for the new feature
- [x] A storybook story has been created or updated
  - Example of SlideTutorial [component](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js) with docgen comments, and its [story](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/docs/other-slidetutorial--default)

### CODE REVIEW

**Comment by goplayoutside3** [comment]
A couple bugs I found in this branch:
1. The SlideTutorial story does not display its markdown content.
2. While a rare scenario, a project team can publish a tutorial that has blank markdown. If for some reason the `activeStep` has no content, the steps following it will not load properly on this branch. Try it out by stepping through the tutorial at  https://local.zooniverse.org:3000/projects/goplayoutside3/test-project-with-translations/classify/workflow/3540. I’ve numbered the step image and step markdown to make the bug obvious. I suggest adding a unit test in SlideTutorial.spec for a case when `activeStep` has no markdown.

File: packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js [hunk 1]
```diff
@@ -83,8 +84,7 @@ function SlideTutorial({
           <Heading level='3' margin={{ bottom: 'xsmall', top: 'small' }}>
             {t('SlideTutorial.heading', { projectDisplayName })}
           </Heading>}
-        {/* TODO: translation */}
-        <Markdownz>{step.content}</Markdownz>
+        <Markdownz>{strings[`steps.${activeStep}.content`]}</Markdownz>
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js:87]: This should use `stepIndex`, not `activeStep`.

**Comment by eatyourgreens** [comment]
I see the bug. The navigation updates `stepIndex`, not `activeIndex`. The tests probably need to be rewritten, not in Enzyme, so that we can test the `useState` hook.

**Comment by eatyourgreens** [comment]
@goplayoutside3 Good catch on that bug, by the way!

**Review by goplayoutside3 — APPROVED** [review]
Looks good!

**Comment by eatyourgreens** [comment]
Still to do: write some React Testing Library tests that mount the tutorial with a mock API.

**Pull request merged**